components/button/test/button-copy.test.js

```diff
@@ -0,0 +1,66 @@
+import '../button-copy.js';
+import { clickElem, expect, fixture, html, nextFrame, oneEvent, runConstructor } from '@brightspace-ui/testing';
+import { stub } from 'sinon';
+
+describe('d2l-button-copy', () => {
+
+	describe('constructor', () => {
+
+		it('should construct', () => {
+			runConstructor('d2l-button-copy');
+		});
+
+	});
+
+	describe('events', () => {
+
+		it('dispatches click event when clicked', async() => {
+			const el = await fixture(html`<d2l-button-copy></d2l-button-copy>`);
+			setTimeout(() => clickElem(el));
```

Comment:
I don't believe the `setTimeout` should be necessary since it's going to happen in another thread anyway.